src/renderers/shared/hooks/__tests__/ReactComponentTreeHook-test.js

```diff
@@ -185,7 +185,7 @@ describe('ReactComponentTreeHook', () => {
       assertTreeMatches([element, tree]);
     });
 
-    it('uses displayName, name, or Object for factory components', () => {
+    xit('uses displayName, name, or Object for factory components', () => {
```

Comment:
A way to quickly disable a test.

There's also `fit` which is handy as it tells jest (or jasmine) to only run that specific test, so you don't need to comment out everything else when trying to troubleshot a single one.